system_tests/speech.py

```diff
@@ -37,7 +37,7 @@ class Config(object):
 
 
 def setUpModule():
-    Config.CLIENT = speech.Client()
+    Config.CLIENT = speech.Client
```

Comment:
Don't do this. You should add a boolean to `Config` that allows you to tell if using gRPC / GAX:

```python
Config.CLIENT = speech.Client()
Config.USE_GAX = Config.CLIENT._use_gax
```

then you can use `unittest.TestCase.skipTest` ([e.g.][1]) to avoid tests that can't run without GAX.

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/319fd8f7e8a77a899747911bdd0b9dc9c357bb2f/system_tests/bigtable.py#L313-L318